PosixPath generator not working for generate_movies python 3.13

These lines of code here:

```
                # Test PosixPath generator for making movie
                file_generator = basepath.glob('test_contour[!_]*.png')
>               result = act.utils.generate_movie(file_generator, write_filename=write_filename)
```

produce an error in the 3.13 testing:

```
E           TypeError: listdir: path should be string, bytes, os.PathLike, integer or None, not map
```

We will need to investigate on what changed

--- a/tests/utils/test_io_utils.py
+++ b/tests/utils/test_io_utils.py
@@ -253,8 +253,9 @@ def test_generate_movie():
             assert np.isclose(Path(write_filename).stat().st_size, 173189, 1000)
 
             # Test PosixPath generator for making movie
-            file_generator = basepath.glob('test_contour[!_]*.png')
-            result = act.utils.generate_movie(file_generator, write_filename=write_filename)
+            result = act.utils.generate_movie(
+                list(basepath.glob('test_contour[!_]*.png')), write_filename=write_filename
+            )
             assert result == str(write_filename)
             assert np.isclose(Path(write_filename).stat().st_size, 173189, 1000)
 

